### PR TITLE
Fix/overview card icon color

### PR DESCRIPTION
**Pull request by obrucheoghene**
## Description
Apply color to OverviewCard component Icon

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?
Using storybook

## Checklist:

- [x] I have performed a self-review of my code

### CODE REVIEW

File: mods/webui/src/stories/overviewcard/OverviewCard.stories.ts [hunk 1]
```diff
@@ -77,7 +79,11 @@ export const Example: Story = {
 export const ChangeIcon: Story = {
   args: {
     label: "Workspace setting",
-    icon: React.createElement(Icon, { name: 'Search', fontSize: "small" })
+    icon: React.createElement(Icon, {
+      name: "Search",
+      fontSize: "small",
+      sx: { color: "#008751" }
```
Comment (psanders) [inline mods/webui/src/stories/overviewcard/OverviewCard.stories.ts]: It feels like this (the color) could be part of the component itself.

Thoughts?

Comment (obrucheoghene) [reply mods/webui/src/stories/overviewcard/OverviewCard.stories.ts]: hmmm, Here the icon props takes in the component that requires the color
I will attempt adding the color to the parent component to see if that works as well

Comment (obrucheoghene) [reply mods/webui/src/stories/overviewcard/OverviewCard.stories.ts]: @psanders 
It worked. 🔥

**Comment by psanders** [comment]
@obrucheoghene 

All good here. Merged!

**Pull request merged**